### ISSUE TITLE
Screen reader stays mute upon activating the 'Selection support pressable' button present under 'A virtualized list with multiple selection support' heading.

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#657]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Narrator

Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to 'All samples' button and invoke it.
Navigate to the 'Virtualized list' item and invoke it.
Navigate to the 'Selection support' button and invoke it.
Now listen to the screen reader announcement.
Actual Result:
Narrator stays mute upon activating the 'Selection support pressable' button present under 'A virtualized list with multiple selection support' heading.
 
Expected Result:
Upon activating the 'Selection Support Pressable' button under the 'A virtualized list with multiple selection support' heading, Narrator should announce the button's label and state change (e.g., "Selection support: Multiple/Single") clearly to the user.
 
User Impact:
User with visual impairments who rely on screen reader will not receive any auditory feedback when interacting with the 'Selection Support Pressable' button. This creates confusion about whether the action was registered, leading to poor usability, reduced confidence in navigation, and potential exclusion for users

MAS Reference:
[MAS 4.2.1 – Object Information](https://aka.ms/MAS4.2.1)

Trap ID:
[1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
